README.md

```diff
@@ -34,6 +48,7 @@ us a DM or [create an issue](https://github.com/EngoEngine/engo/issues/new).
 Engo is currently undergoing a lot of optimizations and constantly gets new features. However, this sometimes means things break. In order to make transitioning easier for you, 
 we have a list of those changes, with the most recent being at the top. If you run into any problems, please contact us at [gitter](https://gitter.im/EngoEngine/engo). 
 
+* `engo` has been split in `engo` (which contains stuff about creating windows, starting the game, creating an OpenGL context, input handling, etc.) - and `core` (which contains a lot of common `System` implementations for common tasks (`RenderSystem`, `CameraSystem`, `AudioSystem`, etc.)
```

Comment:
We should probably explain this change at the top of the README so people understand the difference between `core` and `engo`.

> The top level `engo` package contains the functionality of creating windows, starting the game, working with an OpenGL context and handling input. It is deisgned to be used with Systems designed as per `engo.io/ecs` specifications. In `core` there are our ECS implementations of common game development Systems like a `RenderSystem`, `CameraSystem`.

Obviously that is a really rough draft, and you *could* probably copy and paste what you already have...